### PR TITLE
Remove /jsonapi namespace

### PR DESCRIPTION
**Pull request by cherewaty**
Now that `ui-eholdings` exclusively uses the new JSON API endpoints, we no longer need them sandboxed under `/jsonapi/`.

The older endpoints that simply proxied requests to the EBSCO RM API can now be found under `/proxy/`.

### CODE REVIEW

File: ModuleDescriptor.json
Comment (sadatay) [inline ModuleDescriptor.json:34]: the routes below these should probably be adjusted for the new proxy endpoints, if we keep them at all

**Review by sadatay — CHANGES_REQUESTED** [review]
we'll also want to make sure we don't merge this until the corresponding UI work has been merged - either that or we do some kind of cutover where we keep both the `/jsonapi` prefixed endpoints and the new endpoints around until the UI is adjusted

File: ModuleDescriptor.json
Comment (cherewaty) [reply ModuleDescriptor.json:34]: Routes in module descriptor updated to include `proxy`.

**Review by sadatay — APPROVED** [review]
:shipit:

**Pull request merged**
